envi-tools/include/envitools/ContrastMetrics.hpp

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include <opencv2/core.hpp>
+
+#include "envitools/Box.hpp"
+
+namespace envitools
+{
+class ContrastMetrics
```

Comment:
Just make this a namespace. Doesn't need to be a class since it only holds static functions.